nessie-iceberg/settings.gradle.kts

```diff
@@ -40,56 +40,45 @@ pluginManagement {
     }
   }
 
-  plugins {
+  val versionErrorPronePlugin = "2.0.2"
+  val versionIdeaExtPlugin = "1.1.6"
+  val versionJandexPlugin = "1.82"
+  val versionNessiePlugins = "0.2.12"
+  val versionQuarkus = "2.13.0.Final"
+  val versionShadowPlugin = "7.1.2"
+  val versionSpotlessPlugin = "6.11.0"
```

Comment:
is there no way for these to reuse version mapping from root settings?

Reply:
Those will be managed externally via the version-catalog PR

Reply:
Eventually get rid of those entirely at some time